build/salva2d/Cargo.toml

```diff
@@ -36,21 +36,21 @@ required-features = [ "dim2" ]
 approx = "0.5"
 num-traits = "0.2"
 fnv = "1.0"
-itertools = "0.12"
+itertools = "0.13.0"
```

Comment:
maybe it would be better if you use same pattern, eg. 0.13 instead of 0.13.0. Same for others, not use minor version.

Reply:
Reason for this is that this way this update will last longer. Eg. even if a newer version of rapier, eg. 0.20.1 appears, it will still work with this version of salva. Same for parry, etc.

Reply:
Alright, fixed. 👍